en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
 # Create Your First Service
 
-To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.
+To learn how to create a service, let's look at a simple scenario. In this example, you'll create a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
 
 ## Step 1: Create the service
 
-To create the service, follow the procedure below: 
+To create the service, follow this procedure: 
 
-1. Access the Choreo Console via https://console.choreo.dev/.
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
     
-2. Sign in using either your Google or GitHub credentials.
-    
-3. In the **Services** page, click **Create**. You are directed to the low-code view.
+2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
 
-4. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
 
-5. Select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
+4. Select **GET** as the HTTP method and enter `activecases` as the relative path from the host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
-    
-    Then click **Save API**. Now you have saved the API trigger that starts your service.
-    
-6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+     ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+
+5. Click **Save API**. Now you have saved the API trigger that starts your service.
+
+6. Click **API Calls** and then complete the following steps to configure the API trigger to use the HTTP connection that connects the public API used in this scenario to it:
 
     1. Click **HTTP**.
     
         ![Select HTTP Connection](../assets/img/services/select-http-connection.png)
-        
-    2. Enter information as follows:
-    
-        | **Field**           | **Value**                    |
-        |---------------------|------------------------------|
-        | **Connection Name** | `httpEndpoint`               |
-        | **URL**             | `"https://api.covid19api.com"` |
-        
-        Click **Save**.
-        
+         
+    2. Enter the following information and then click **Save**:
```

Comment:
The way we present input value, field vs value is different from place to place. shall we try to follow one approach?

Reply:
The plan is to suggest the use of backticks for user input values and bold formatting for field names.